### ISSUE TITLE
error compiling

### ISSUE THREAD

**Issue by hennonoman** [oddbit/flutter_facebook_app_events#74]
Hello,
I got this error after added the package in pubspec.yaml

error: compiling for iOS 8.0, but module 'FBSDKCoreKit' has a minimum deployment target of iOS 9.0

**Comment by charliegaldamez** [comment]
For now, I could resolve this by adding this to the `Podfile` 

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '9.0'
    end
  end
end
```

Hope it can work for you, too.

**Comment by DennisAlund** [comment]
For further reference, also please use issue template so that it's easier for us to get all the necessary information 🙏 

![image](https://user-images.githubusercontent.com/2051607/107303079-5acfe480-6ab9-11eb-9012-c62b4a6a323b.png)

**Comment by DennisAlund** [comment]
Please let me know if this is related to the same discussion as in https://github.com/oddbit/flutter_facebook_app_events/issues/64#issuecomment-742910592

**Comment by DennisAlund** [comment]
Fixed in #72
